_includes/embed/video.html

```diff
@@ -0,0 +1,24 @@
+{% assign url = include.src %}
+
+{% unless url contains '://' %}
+  {%- capture url -%}
+    {% include img-url.html src=include.src img_path=page.img_path absolute=false %}
```

Comment:
```suggestion
    {% include img-url.html src=include.src img_path=page.img_path %}
```

Reply:
The default is false, there is no need to set it explicitly.

Reply:
For this new feature, we also need to add instructions in "writing a new post".

Reply:
> For this new feature, we also need to add instructions in "writing a new post".

I agree. The reason I postponed this is because I need to have a video file to use. Is it possible to get some at https://chirpy-img.netlify.app? Alternatively, I can just add instructions without having a live demo.

Reply:
> Is it possible to get some at <https://chirpy-img.netlify.app/>

OK, that's another hidden repository for demo site image resources, I'll add you as a collaborator later.

Reply:
Local video files may not be necessary to upload to the image repository, as you say, only add instructions for embedding local videos in `Writing a New Post`, without having to add more video examples in `Text and Typography`.

Reply:
That's what I was going to suggest later today. It turns out it's not so easy to find proper video example to be used for a demo considering rights and file size. However, it was a good thing to elaborate as I have decided to add support for `poster` in addition to other attributes that may be useful in production.

I'm on it.